.github/workflows/ci.yml

```
@@ -57,6 +68,45 @@ jobs:
       - name: Ruff format
         run: ruff format --check .
 
+  typecheck:
```

actionlint (1.7.12)

[error] "runs-on" section is missing in job "typecheck"

(syntax-check)